tests/testsuite/read_manifest.rs

```diff
@@ -156,25 +248,218 @@ fn cargo_read_manifest_with_specified_readme() {
         .build();
 
     p.cargo("read-manifest")
-        .with_json(&manifest_output(&format!(r#""{}""#, "SomeReadme.txt")))
+        .with_stdout_data(
+            str![[r#"
+{
+  "authors": [
+    "wycats@example.com"
+  ],
+  "categories": [],
+  "default_run": null,
+  "dependencies": [],
+  "description": null,
+  "documentation": null,
+  "edition": "2015",
+  "features": {},
+  "homepage": null,
+  "id": "path+[ROOTURL]/foo#0.5.0",
+  "keywords": [],
+  "license": null,
+  "license_file": null,
+  "links": null,
+  "manifest_path": "[ROOT]/foo/Cargo.toml",
+  "metadata": null,
+  "name": "foo",
+  "publish": null,
+  "readme": "SomeReadme.txt",
```

Comment:
Should we `"...": "{...}" ` all of the other keys for each of these `"readme"` tests?

Reply:
Redacted!